kore/test/Test/Kore/Builtin/Builtin.hs

```diff
@@ -216,7 +216,7 @@ runStep
     -- ^ axiom
     -> IO
         (Either
-            (UnificationOrSubstitutionError Variable)
+            UnificationOrSubstitutionError
```

Comment:
Since you changed other stuff to one-liners: the `Either` also fits on one line now. Not sure if it's better, but I thought I'd mention it.